### DISCUSSION TITLE
node14

### DISCUSSION THREAD

**Discussion by EricChen047** [Homebrew/discussions#95] [Everyday usage]
Hi, why there's no node@14? Please~~

**Comment by SMillerDev** [comment]
Because nobody put in the work yet.

**Comment by Rylan12** [comment]
Work is in progress in https://github.com/Homebrew/homebrew-core/pull/63410. Follow that thread for updates.

In general, though, the best way to get a formula added to Homebrew/core is to open a PR that adds it.
